extensions/wikia/AffiliateService/AffiliateService.i18n.php

```diff
@@ -0,0 +1,58 @@
+<?php
+$messages = array();
```

Comment:
I would consult confluence (maybe: https://wikia-inc.atlassian.net/wiki/spaces/PLAT/pages/110593849/i18n) or ping the engineering channel to determine the conventions for these files.

Reply:
Following the convention of other i18n files in the repo including Trivia Quizzes.